### PR TITLE
Add the Maintainership article

### LINKED ISSUE #277: Add information about maintainer files to Packaging documentation
The current documentation on submitting an update for a package does not mention maintainer files. In particular there doesn't appear to be indicators that only the maintainers can submit a patch outside of the maintainer file itself.

I would probably suggest an addition the "Prior to Patch Submission" category found [here](https://getsol.us/articles/packaging/submitting-a-package/en/) . It is a bit frustrating to put work into a patch only to later learn that it can't be used.  I understand why the core team would rather packages be updated by their maintainer but it appears to have happened a few times over the years so probably wouldn't hurt.

### PR DESCRIPTION
**Pull request by livingsilver94**
## Description

Add the Maintainership article.
Fixes #277.

### Submitter Checklist

- [x] Updated the "lastmod" portion at the top of any modified Markdown files.
- [ ] Squashed commits with `git rebase -i` (if needed)

### CODE REVIEW

**Comment by livingsilver94** [comment]
Again, I'm not a native English speakers so some phrases may sound non-idiomatic or even be wrong.

Other articles in the packaging section should link to the Maintainership article. I did not do that because I think it has to be discussed. packaging-practices, request-a-package-update, submitting-a-package and updating-an-existing-package are good candidates, although the may be too many. Just let me know where you want it linked.

File: packaging/maintainership/en.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+## Stepping in
+
+To officially step in as the maintainer of a package, a `MAINTAINERS.md` file must be provided for accepted packages that are not yet included in the repository, or if the package predates the policy of requiring one. Instead, if a previously maintained package is marked by the Solus team as needing a new maintainer, the `MAINTAINERS.md` file must be updated.
+
+## Updating a maintained package
+
+The procedure varies depending on whether or not the individual is the maintainer.
+
+### Maintainer individuals
```
Comment (JoshStrobl) [inline packaging/maintainership/en.md:28]: Can probably just be "Maintainers"

File: packaging/maintainership/en.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+Maintainers are free to [update a package](/articles/packaging/updating-an-existing-package/en) at their will, unless otherwise indicated by the Core Team.
+
+### Non-maintainer individuals
```
Comment (JoshStrobl) [inline packaging/maintainership/en.md:32]: Maybe just "Non-Maintainers"

File: packaging/maintainership/en.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+If a package is actively maintained, modifications should not occur and the individual should exercise patience when it comes to the updating of software. In some cases, a maintainer may intentionally be holding back a package, or has simply not updated yet. If pertinent, the individual should file a package update request. Alternatively, the individual can reach the maintainers or the Core Team on IRC and ask permission to update the package. It is also possible to submit a patch and attach a message to it clarifying the intention of updating the package, although this is a special case reserved to e.g. security updates and the patch is likely to be rejected.
```
Comment (JoshStrobl) [inline packaging/maintainership/en.md:34]: Let's get package update request linked. `[package update request](/articles/packaging/request-a-package-update/en)`

File: packaging/maintainership/en.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+## Template
+
+Presented here is the `MAINTAINERS.md` file. This file must be provided verbatim alongside the other patch contents, and filled in with the maintainers' personal information. An IRC contact is optional but recommended, but an e-mail address is mandatory. For accountability purposes, packagers must also fill in their real names.
```
Comment (JoshStrobl) [inline packaging/maintainership/en.md:38]: "For accountability purposes, packagers must also fill in their real names."

Let's clarify it further with the content below, just so everyone is on the same page that it is already required as part of the .solus/packager file:

```
Similar to the `.solus/packager` file used for packaging, the maintainers listed in `MAINTAINERS.md` must use their real first and last name(s) for accountability purposes.
```

File: packaging/maintainership/en.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+# Maintainership
+
+The role of a maintainer is to update package(s) on a regular basis, while following Solus packaging and consistency standards.
+
+## Commitment
+
+Each new package which is going to land in the Solus repository must have one or more maintainers. These individuals commit to:
+ - Update the package reasonably regularly, unless there are valid reasons to hold it back
+ - Ensure the [packaging file](/articles/packaging/package.yml/en) adheres to the Solus [standards](/articles/packaging/packaging-practices/en)
+ - Ensure the application or library is consistent with the Operating System aesthetics, file system conventions and the Solus philosophy in general
```
Comment (JoshStrobl) [inline packaging/maintainership/en.md:14]: Newline above first item, remove one space indent.

**Review by JoshStrobl — CHANGES_REQUESTED** [review]
Otherwise LGTM, thanks!

File: packaging/maintainership/en.md (hunk 4 above)
Comment (livingsilver94) [reply packaging/maintainership/en.md:38]: `.solus/packager` or `~/.config/whatever` ?

Comment (JoshStrobl) [reply packaging/maintainership/en.md:38]: Our Building a Package doc references .solus/packager so let's stick with that for now.

**Comment by JoshStrobl** [comment]
LGTM, thanks!

**Pull request merged**